app/views/passwords/edit.html.erb

```diff
@@ -0,0 +1,18 @@
+<div id="clearance" class="password-reset">
```

Comment:
Are the `password/` files generated by clearance? If so I think we can delete them